charts/enterprise-kyverno-operator/templates/_cr-kyverno-helpers.tpl

```diff
@@ -37,6 +37,14 @@ content:
       {{- toYaml .Values.kyverno.helm | nindent 6 }}
       {{- end}}
       admissionController:
+        podLabels: 
+{{- if .Values.globalLabels }}
+{{- toYaml .Values.globalLabels | nindent 10 }}
+{{- end}}
+        podAnnotations: 
+{{- if .Values.globalAnnotations }}
+{{- toYaml .Values.globalAnnotations | nindent 10 }}
+{{- end}}
```

Comment:
Can we confirm if the globalLabels and globalAnnotations are "added" to default labels and annotations in kyverno pods? Or it completely overwrites? If it overwrites, there might be other issues, so we better avoid that.

Reply:
Confirmed!